### PR TITLE
Test with GDAL<3 PROJ 6

### PR DESCRIPTION
**Pull request by snowman2**
@conda-forge-admin please rerender.

### CODE REVIEW

**Comment by snowman2** [comment]
@conda-forge-admin please rerender.

**Comment by snowman2** [comment]
@ocefpaf, looks like its working. Any issues merging if all builds pass?

**Comment by snowman2** [comment]
Looks like all tests pass.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,12 +23,11 @@ requirements:
     - pip
     - cython
     - numpy
-    - libgdal
-    - proj4 <6
+    - libgdal <3
```
Comment (ocefpaf) [inline recipe/meta.yaml]: If you want to merge this just change `<3` to `2.4` and re-render. I guess that having `proj 6` is better than `gdal 3` at this point.

**Comment by snowman2** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (snowman2) [reply recipe/meta.yaml]: Sounds good. Will do. Thanks!

**Comment by ocefpaf** [comment]
Thanks!

**Comment by snowman2** [comment]
Just FYI @djhoese

**Comment by snowman2** [comment]
Not a problem. Glad it finally works! This is the last piece for the conda-forge python community to use PROJ 6.

**Comment by ocefpaf** [comment]
> This is the last piece for the conda-forge python community to use PROJ 6.

`cartopy` is still wobbly though :-/

https://github.com/conda-forge/cartopy-feedstock/pull/64

**Pull request merged**